openmdao/docs/usr-guide/tutorials/paraboloid-tutorial.rst

```diff
@@ -257,6 +257,42 @@ The output should look like this:
 
    -15.0
 
+The `IndepVarComp` component is used to define a source for a parameter that
+we want to use as a design variable for a driver. In our case, we want to
+optimize the Paraboloid model, finding values for 'x' and 'y' that minimize
+the output 'f_xy.' Sometimes we just want to run our component once to see
```

Comment:
maybe a paragraph break before Sometimes?  it kind of runs together... separating the concepts might make it easier to read